checklist/checklist_sys.csv/tests.csv

```diff
@@ -0,0 +1,9 @@
+ID,Topic,Title,Requirement,Explanation,References
+2.1,Data Presence,Test Data Fetching and File Reading,"Verify that the data fetching API or data file reading functionality works correctly. Ensure that proper error handling is in place for scenarios such as missing files, incorrect file formats, and network errors.","Ensure that the code responsible for fetching or reading data can handle errors. This means if the file is missing, the format is wrong, or there's a network issue, the system should not crash but should provide a clear error message indicating the problem.",(general knowledge)
+3.1,Data Quality,Validate Data Shape and Values,"Check that the data has the expected shape and that all values meet domain-specific constraints, such as non-negative distances.","Check that the data being used has the correct structure (like having the right number of columns) and that the values within the data make sense (e.g., distances should not be negative). This ensures that the data is valid and reliable for model training.","alexander2024Evaluating, ISO/IEC5259"

```

Comment:
This does seem important, but I wonder does this kind of test go in the `tests` folder, or is this part of the EDA script in a data analysis? This seems a different class of tests compared to the tests above which are testing the code for retrieving and saving data. This makes me think about the test pyramid in Openja et al... This is not a negative comment, just a question to drive future discussion.

Reply:
Hi Tiffany. 
I think EDA and data wrangling are to explore and transform the data, 
while the test case here is to validate the raw/processed data after a bunch of operations, to ensure the data is as expected for the next stage (e.g. model training)

Reply:
Hi @ttimbers 
Indeed we also have a question whether this Checklist item shall target raw data or processed data?
To me, targeting raw data implies a validation on the data source, while targeting processed data implies a validation on the pre-processing operation by the user.